extensions/objects/templates/FilepathEditorDirective.ts

```diff
@@ -17,6 +17,7 @@
  */
 
 // This directive can only be used in the context of an exploration.
+require('components/forms/custom-forms-directives/image-uploader.directive.ts');
 require('services/CsrfTokenService.ts');
```

Comment:
Please add requires for the other dependencies like `ContextService`

Reply:
Done